src/react-medium-image-zoom.js

```diff
@@ -266,16 +274,19 @@ class Zoom extends Component {
   }
 
   render() {
+    const { defaultStyles, hasAlreadyLoaded, zoomImage } = this.props
+    const { isZoomed, src } = this.state
+
     return (
       <div onClick={ this.handleUnzoom } style={ this.getZoomContainerStyle() }>
         <Overlay
-          isVisible={ this.state.isZoomed }
-          defaultStyles={ this.props.defaultStyles }
+          isVisible={ isZoomed }
+          defaultStyles={ defaultStyles }
         />
         <img
-          src={ this.state.src }
-          alt={ this.props.alt }
-          className={ this.props.className }
+          { ...zoomImage }
+          src={ src }
+          srcSet={ hasAlreadyLoaded ? zoomImage.srcSet : null }
```

Comment:
I've noticed that the `srcset` attribute hasn't been getting added on the first load in the example -- is this logic backwards by chance?